src/lib/parser.ts

```diff
@@ -75,7 +76,11 @@ export function parseOutput(gptString: string): ParsedOutput {
       .forEach((line: string) => {
         try {
           commands.push(parseFunctionCall(line));
-        } catch (e) {}
+        } catch (e) {
+          // In case the output has the "tell user" section under
+          // "commands".
+          unparsedCommands.push(line.replace("tell user:", ""));
```

Comment:
Can you remember, when we saw this bug, was it on the old fine-tuned model?

This is a tricky one because it semi-often puts other irrelevant text under commands.

I've seen `None` and `# Some comment about the commands` a few times and never seen the tell user/commands bug

Reply:
it only gets displayed if there's no "tell user:" field through 🤔  Have you seen it output only nonsense under Commands + no "tell user" section?